org-formation/710-tgw/_tasks.yaml

```diff
@@ -43,6 +43,7 @@ TransitGateway:
       - !Ref ScipoolDevAccount
       - !Ref ScipoolProdAccount
       - !Ref SageITAccount
+      - !Ref DnTDevAccount
```

Comment:
Our application will be deployed on `DpeProdAccount`.

Reply:
fixed